diff --git a/ios/LinkPreview/LinkPreviewNativeActionView.swift b/ios/LinkPreview/LinkPreviewNativeActionView.swift
index 233b524..9777444 100644
--- a/ios/LinkPreview/LinkPreviewNativeActionView.swift
+++ b/ios/LinkPreview/LinkPreviewNativeActionView.swift
@@ -137,7 +137,7 @@ class LinkPreviewNativeActionView: RouterViewWithLogger, LinkPreviewMenuUpdatable {
     baseUiAction.attributes = attributes
     baseUiAction.state = isOn == true ? .on : .off
 
-    if let subtitle = subtitle {
+    if #available(iOS 16.0, *), let subtitle = subtitle {
       baseUiAction.subtitle = subtitle
     }
     if let label = discoverabilityLabel {
